changelog/_unreleased/2020-10-15-fix-filter-aggregation-in-javascript-criteria.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Fix filter aggregation in JavaScript criteria
+author: Marvin Schröder
+---
+# Administration
+* Fix filter aggregation in the JavaScript criteria class
```

Comment:
There are some keywords which are used to describe changes, have a look at the [CONTRIBUTING.md](https://github.com/shopware/platform/blob/master/CONTRIBUTING.md#contributing-to-the-shopware-code-base) and change this entry accordingly please.